packages/storybook/stories/additional-docs.js

```diff
@@ -46,22 +46,10 @@ export const additionalDocs = {
     maturityCategory: USE,
     maturityLevel: DEPLOYED,
   },
-  'Banner - Maintenance': {
-    maturityCategory: USE,
-    maturityLevel: DEPLOYED,
-    guidanceHref: 'banner/maintenance',
-    guidanceName: 'Banner - maintenance',
-  },
   'Modal - React': {
     maturityCategory: DONT_USE,
     maturityLevel: DEPRECATED,
   },
-  'OMB info - React': {
-    guidanceHref: 'omb-info',
-    guidanceName: 'OMB info',
-    maturityCategory: DONT_USE,
-    maturityLevel: DEPRECATED,
-  },
```

Comment:
I am intentionally keeping these components removed from Storybook so that they at lest won't appear in our documentation any longer.